benchmarks/src/main/java/com/google/samples/apps/nowinandroid/interests/InterestsActions.kt

```diff
@@ -48,3 +48,17 @@ fun MacrobenchmarkScope.interestsToggleBookmarked() {
     checkable.click()
     device.waitForIdle()
 }
+
+fun MacrobenchmarkScope.setAppTheme(isDark: Boolean) {
+    when (isDark) {
+        true -> device.findObject(By.text("Dark")).click()
+        false -> device.findObject(By.text("Light")).click()
+    }
+    device.waitForIdle()
+    device.findObject(By.text("OK")).click()
+
+    // Wait until interests are shown on screen
```

Comment:
It looks like this code is waiting for the top app bar to be shown, rather than any interests/topics.